src/config.ts

```diff
@@ -0,0 +1,12 @@
+export default {
+  algorithm: 'aes-256-ctr',
+  preloader: {
+    username: 'preloader',
+    address: 'AYeceuGa7tTsyG6jgq7X6qKdoXt9iJJKN6',
+    passphrase: 'window perfect fiber whip source stock tower upset crucial later letter social'
+  },
+  nodes:
+    process.env.NODE_ENV === 'production'
```

Comment:
This is of no use as you have the same URL for all environments. Either remove this or add a different URL.

Reply:
https://github.com/ArkEcosystem/poc-ark-messenger/pull/7